<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rentDialog</class>
 <widget class="QDialog" name="rentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>132</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Date Borrowed</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Return date</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QDateEdit" name="borrowedEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Rent</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
